src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -78,3 +79,39 @@ let DetectAsteriskInPackageReferenceItems1 () =
 let DetectAsteriskInPackageReferenceItems2 () =
     let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyFsprojWithoutAsterisk.fsproj")))
     Assert.That(DetectAsteriskInPackageReferenceItems fileInfo, Is.EqualTo false)
+
+
+[<Test>]
+let EolAtEofTest1() =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithEolAtEof.fsx")))
+    Assert.That(EolAtEof fileInfo, Is.EqualTo True)
+
+
+[<Test>]
+let EolAtEofTest2() =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyWithoutEolAtEof.fsx")))
+    Assert.That(EolAtEof fileInfo, Is.EqualTo False)
+
+
+[<Test>]
+let EolAtEofTest3() =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "someLib.dll")))
+    Assert.That(EolAtEof fileInfo, Is.EqualTo NotApplicable)
+
+
+[<Test>]
+let HasBinaryContentTest1 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "someLib.dll")))
+    Assert.That(HasBinaryContent fileInfo, Is.EqualTo true)
+
+
+[<Test>]
+let HasBinaryContentTest2 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "white.png")))
+    Assert.That(HasBinaryContent fileInfo, Is.EqualTo true)
+
+
+[<Test>]
+let HasBinaryContentTest3 () =
+    let fileInfo = (FileInfo (Path.Combine(__SOURCE_DIRECTORY__, "DummyFiles", "DummyNonBinaryFile.txt")))
+    Assert.That(HasBinaryContent fileInfo, Is.EqualTo false)

```

Comment:
@realmarv tests should use common var dummyFilesDirectory

Reply:
Done